app/services/cache/ExportsCacheService.scala

```diff
@@ -16,25 +16,27 @@
 
 package services.cache
 
-import java.time.Instant
+import java.time.{Clock, Instant}
 
+import connectors.CustomsDeclareExportsConnector
 import javax.inject.{Inject, Singleton}
 import models.ExportsDeclaration
-import reactivemongo.play.json.collection.JSONBatchCommands.FindAndModifyCommand
+import uk.gov.hmrc.http.HeaderCarrier
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class ExportsCacheService @Inject()(journeyCacheModelRepo: ExportsDeclarationRepository)(
+class ExportsCacheService @Inject()(connector: CustomsDeclareExportsConnector)(
```

Comment:
Just to highlight, instead of saving/retrieving from the front end cache, this PR makes each update save to the back end (otherwise we have to persist long lived drafts in the frontend cache which has security implications)